services/core-api/app/api/mines/permits/permit_extraction/tasks.py

```diff
@@ -0,0 +1,23 @@
+
+from app.api.mines.permits.permit_extraction.create_permit_conditions import (
+    create_permit_conditions_from_task,
+)
+from app.api.search.search.permit_search_service import PermitSearchService
+from app.tasks.celery import celery
+
+
+@celery.task(max_retries=360)
+def poll_update_permit_extraction_status(permit_extraction_task_id):
```

Comment:
Introduced this celery task to poll for a status update from the permit service. The actual triggering of the permit extraction task through the API happens within the actual `POST` endpoint